Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabViewportFocusPathHandler.cpp

```diff
@@ -85,6 +91,30 @@ namespace AzToolsFramework::Prefab
         Refresh();
     }
 
+    void PrefabViewportFocusPathHandler::OnEditorModeActivated(
+        [[maybe_unused]] const ViewportEditorModesInterface& editorModeState, ViewportEditorMode mode)
+    {
+        // This function triggers when the Editor changes from regular editing to a Component Mode.
+        if (mode == ViewportEditorMode::Component)
+        {
+            // Disable the widgets to prevent it from being used to change selection.
+            m_backButton->setEnabled(false);
```

Comment:
Nit:  You could move these two calls to a method called something like "EnableBreadcrumbWidgets(bool enable)" in case it grows in the future, etc.  Just to avoid having to update multiple locations if another widget gets added to the list.  Just noticed they were repeated in both methods with just the true/false difference

Reply:
Good point! That widget is legacy and we will probably red-code it soon, but I'll keep that in mind if that changes or I encounter similar cases :D